src/components/TypesOfRecipeCards/RecipeCard.jsx

```diff
@@ -0,0 +1,88 @@
+import { PushPin } from "@phosphor-icons/react";
+
+//RecipeCard is now able to take in recipe and review, and render the correct information
+
+const RecipeCard = ({ pinned, recipe }) => {
+  const white = "#d4d4d4";
+  const size = 18;
+  const orginalRecipe = recipe.recipeId ? recipe.recipeId : recipe;
```

Comment:
Why do you need this? I also don't understand why it's "original"

Reply:
I tried to change it to just recipe is does not work

Reply:
that seems strange... is this a mongoose thing?

Reply:
nvm!

Reply:
still don't understand. looks like you just renamed the variable?

Reply:
but now it's "recipe"? This component is acting as two cards. Maybe I shouldn't have done that. If it has recipeId it will act as a comment card, if it doesn't it will act as a recipe card.

Reply:
still don't understand. don't you have a different component for comment card?

Reply:
I probably should've named it differently
comment card -  is actually for comments under the recipe 
this comment is the "comment" or review that the user made for a particular recipe

Reply:
That sounds really confusing. When I read the code, I wouldn't know it's a review vs recipe. If you think they can be the same component, then you should rename the rename the variable eg. `isReview`